oap-server/server-core/src/main/java/org/apache/skywalking/oap/server/core/storage/PersistenceTimer.java

```diff
@@ -43,16 +48,18 @@
 @Slf4j
 public enum PersistenceTimer {
     INSTANCE;
-    private Boolean isStarted = false;
+    @VisibleForTesting
```

Comment:
> 2021-06-24T13:37:45.9314106Z [ERROR] Failed to execute goal org.apache.maven.plugins:maven-compiler-plugin:3.8.0:testCompile (default-testCompile) on project server-core: Compilation failure
2021-06-24T13:37:45.9317124Z [ERROR] /home/runner/work/skywalking/skywalking/oap-server/server-core/src/test/java/org/apache/skywalking/oap/server/core/storage/PersistenceTimerTest.java:[80,34] isStarted has private access in

Is this annotation really working?

Reply:
It just means that I expect him to be used for testing, it has no practical meaning

Reply:
Then don't remove `private`, you should use `Whitebox` for tests.